keel-web/src/test/resources/application.properties

```diff
@@ -14,3 +14,5 @@ sql.connection-pools.default.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:somepo
 sql.migration.jdbc-url=jdbc:tc:mysql:5.7.22://somehostname:someport/databasename
 spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
 spring.application.name=keel
+keel.plugins.ec2.enabled=true
+keel.plugins.titus.enabled=true
```

Comment:
There are a few other tests where we explicitly enable these. Maybe we can not bother doing so any more?